src/main/java/edu/oregonstate/cs361/battleship/Main.java

```diff
@@ -778,18 +789,79 @@ public static boolean Hit(Point shipStart, Point shipEnd, Point shotPoint){
                     return true;   // if the coordinates of current point match shot, you hit!
             }
 
-            return false; // check all points ship lies on, found no match to shot point
+            //return false; // check all points ship lies on, found no match to shot point
 
-        } else if (shotPoint.getAcross() == shipStart.getAcross()) { // if start and end on same x coordinate, ship is vertical
+        } else if (shipStart.getAcross() == shipEnd.getAcross()) { // if start and end on same x coordinate, ship is vertical
             int x = shipStart.getAcross();
             for (int y = shipStart.getDown(); y <= shipEnd.getDown(); y++) {
 
                 if (x == shotPoint.getAcross() && y == shotPoint.getDown())
                     return true;   // if the coordinates of current point match shot, you hit!
             }
-            return false; // check all points ship lies on, found no match to shot point
+            //return false; // check all points ship lies on, found no match to shot point
         }
 
         return false; // points given are not horizontal or vertical and not valid, can't hit diagonally
     }
+
+    /*public static BattleshipModelUpdated Sink( Point shipStart, Point shipEnd, boolean player, BattleshipModelUpdated model ){
```

Comment:
pseudo code for sinking and will either add it or delete it on the final. This is what your issue is about.